lib/use-checkout-pricing.js

```diff
@@ -38,20 +47,39 @@ export default function useCheckoutPricing(initialInputs = {}, handleError = thr
   const recurly = useRecurly();
   const [loading, setLoading] = useState(false);
   const [input, setInput] = useState(initialInputs);
-  let checkoutPricing = recurly.Pricing.Checkout();
-  const [pricing, setPricing] = useState(checkoutPricing);
+  const [pricing, setPricing] = useState(recurly.Pricing.Checkout());
 
   useEffect(() => {
     setLoading(true);
-    const { subscriptions = [] } = input;
+    let checkoutPricing = recurly.Pricing.Checkout();
+    const { subscriptions = [], ...singleInputs } = input;
+
+    checkoutPricing = addSingleInputs(singleInputs, checkoutPricing);
+
     addSubscriptions(subscriptions, checkoutPricing)
       .then(() => {
-        setPricing(checkoutPricing);
+        checkoutPricing = checkoutPricing.reprice().done(() => {
```

Comment:
`done` will call `reprice`: https://github.com/recurly/recurly-js/blob/master/lib/recurly/pricing/promise.js#L52

Reply:
I tried to make this change, but am getting the error `checkoutPricing.done is not a function` without the `.reprice()` call before it.

Reply:
Ah! I missed that it would not be a `PricingPromise` at this time. carry on.